src/components/Settings/AccountTab.react.js

```diff
@@ -191,8 +195,11 @@ class AccountTab extends React.Component {
   };
 
   handleAvatarSubmit = async () => {
-    const { file } = this.state;
+    const { imageFile } = this.state;
     const { accessToken, actions, userEmailId } = this.props;
+    const fileExt = extractImageFileExtensionFromBase64(imageFile);
+    const fileName = 'image.' + fileExt;
+    const file = base64StringtoFile(imageFile, fileName);
```

Comment:
@akshatnitd I have converted the image data(croppedImageData) which I get in `imageFile` in `base64` format to file format.
Then appended it using `form.append(''image",file)`. And the croppedImage is getting saved successfully.